test/components/remote_retro_test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { shallow } from "enzyme"
+import { expect } from "chai"
+import RemoteRetro from "../../web/static/js/components/remote_retro"
+
+describe("<RemoteRetro>", () => {
+  const mockRetroChannel = {
+    on: () => {},
+    join: () => {},
+  }
+  context("when the current user is a facilitator", () => {
+    const presences = {
+      userToken: {
+        user: { facilitator: true },
+      },
+    }
+
+    it("renders a room with isFacilitiator true", () => {
+      const wrapper = shallow(
+        <RemoteRetro userToken="userToken" retroChannel={mockRetroChannel} />)
+
+      wrapper.setState({ presences })
```

Comment:
respect.

looking at how presence gets set, you could mock `props.retroChannel.on`, and call the second arg with presences. that said, thinking about a longer term approach sounds like a better idea. you the man, @crq. don't let anyone tell you otherwise.